lib/cesium_link/redirects/redirect.ex

```diff
@@ -0,0 +1,37 @@
+defmodule CesiumLink.Redirects.Redirect do
+  @moduledoc """
+  Redirect schema.
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  @required_fields ~w(name slug url type edited_at)a
+  @optional_fields ~w(visits)a
+
+  @primary_key {:id, :binary_id, autogenerate: true}
+  @foreign_key_type :binary_id
+  schema "redirects" do
+    field :name, :string
+    field :slug, :string
+    field :type, Ecto.Enum, values: [:default, :form]
+    field :url, :string
+    field :visits, :integer, default: 0
+    field :edited_at, :utc_datetime
```

Comment:
This one was interesting. 😅
Basically, when updating the view count of the redirect, the `updated_at` column was also being refreshed automatically (which is correct behaviour). The problem is, I wanted to make sure that users could see the last time a redirect was edited and using the `updated_at` field would've yielded incorrect results, because `visits` is not something managed by the user, it's view only and used for metrics. So I ended up creating a specific `edited_at` timestamp column that only gets updated on edit actions through the apps admin dashboard (and this is the one displayed in the redirects table).